src/routes/edit/share-options/exports/frameworks/angular-fragment.ts

```diff
@@ -314,7 +314,7 @@ export const createAngularApp = (fragment: any, fragments: any[]) => {
 			'tslib': '2.3.0',
 			'sass': '1.45.0',
 			'zone.js': '0.11.4',
-			'carbon-components-angular': '4.56.3',
+			'carbon-components-angular': '4.63.0',
 			'carbon-components': '10.50.0'
```

Comment:
```suggestion
			'carbon-components': '10.58.0'
```
Also update `carbon-components` version for react export to `10.58.0` for consistency